src/main/java/frc/robot/Constants.java

```diff
@@ -89,16 +89,19 @@ public static class TurretConstants {
     public static final int DEFAULT_SHOT_ROTATION = 0;
 
     // PID coefficients
-    // TODO: Change values when there's an actual real functional robot.
-    public static final HowdyPID TURRET_PID = new HowdyPID(0.125, 0.001, 0);
+    public static final boolean ADVANCE_LOOP = false;
+    public static final HowdyPID TURRET_POSITION_PID_CASCADE = new HowdyPID(5, 0.01, 0.225, 0.05, 0);
+    public static final HowdyPID TURRET_VELOCITY_PID_CASCADE = new HowdyPID(5, 0.01, 0.225, 0.05, 0);
+    public static final HowdyPID TURRET_POSITION_PID = new HowdyPID(5, 0.01, 0.225, 0.05, 0);
+
 
     public static final double TURRET_KMAXOUTPUT = 1;
     public static final double TURRET_KMINOUTPUT = -1;
 
-    public static final int TURRET_MIN_ANGLE_DEGREES = 0;
-    public static final int TURRET_MAX_ANGLE_DEGREES = 40; // 56.2 - 11 measured
+    public static final int TURRET_MIN_ANGLE_DEGREES = -30;
+    public static final int TURRET_MAX_ANGLE_DEGREES = 30; // 56.2 - 11 measured
 
-    public static final double TURRET_CONVERSION_FACTOR = 0.25;
+    public static final double TURRET_CONVERSION_FACTOR = 0.18292682926829268292682926829268;
```

Comment:
If you want the most accuracy, instead of a long literal you should put the formula you used to get this number here and let the compiler do the math. Also somewhat self-documenting that way as well.